src/network/HTTP.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+export default class HTTP {
+  constructor({ protocol = 'http', hostname = 'localhost', port = 8181, token }) {
+    if (protocol !== 'http' && protocol !== 'https') {
+      throw new Error("Invalid protocol: must be either 'https' or 'http'");
+    }
+
+    this.baseUrl = `${protocol}://${hostname}:${port}`;
+    this.header = {
+      auth_token: token,
```

Comment:
Please, verify if this token is correctly provided.

Reply:
Done!